Source/Commands/LambdaBodyCommand.ts

```diff
@@ -0,0 +1,110 @@
+/// <reference path="Parameters/SingleParameter.ts" />
+/// <reference path="Parameters/RepeatingParameters.ts" />
+
+namespace GLS.Commands {
+    "use strict";
+
+    /**
+     * A command for a lambda function body.
+     */
+    export class LambdaBodyCommand extends Command {
+        /**
+         * Information on parameters this command takes in.
+         */
+        private static parameters: Parameters.Parameter[] = [
+            /*
+            new Parameters.SingleParameter("returnType", "The return type of the function.", true),
+            */
+            new Parameters.SingleParameter(
+                "returnType",
+                "Return type of the lambda function",
+                true),
+            new Parameters.RepeatingParameters(
+                "Lambda function parameters.",
+                [
+                    new Parameters.SingleParameter(
+                        "parameterName",
+                        "A named parameter for the lambda function.",
+                        true),
+                    new Parameters.SingleParameter(
+                        "parameterType",
+                        "The type of the parameter.",
+                        true)
+                ]),
+            new Parameters.SingleParameter(
+                "functionBody",
+                "The actual body of the lambda function",
+                true)
+        ];
+
+        /**
+         * @returns Information on parameters this command takes in.
+         */
+        public getParameters(): Parameters.Parameter[] {
+            return LambdaBodyCommand.parameters;
+        }
+
+        /**
+         * Renders the lambda for a language with the given parameters.
+         *     GLS syntax (order of parameters) lambda returnType [parameterName, parameterType, ...] command
+         *     Render format: lambdaLeft parameterType parameterName, ... lambdaMiddle commandString lambdaRight
+         * @param parameters   The command's name, followed by any parameters.
+         * @returns Line(s) of code in the language.
+         * @remarks Usage: ([parameterName, parameterType, ...]).
+         */
+        public render(parameters: string[]): LineResults {
+            let returnType: string = this.context.convertCommon("type", parameters[2]),
+                lambdaBody: string = "",
+                output: CommandResult[];
+
+            lambdaBody += this.language.properties.lambdas.functionLeft;
+
+            if (this.language.properties.lambdas.returnTypeRequired) {
+                throw Error("returnTypeRequired=true not implemented");
+            }
+
+            if (parameters.length > 3) {
+                lambdaBody += this.generateParameterVariable(parameters, 2);
+
+                // Condition (i + 1) < parameters.length to ensure we don't accidentally
+                // render the command as a variable.
+                for (let i: number = 4; (i + 1) < parameters.length; i += 2) {
```

Comment:
Also super tiny nit: most other commands that do this do `i < parameters.length - 1`... It would be 0.1% nicer if you could do that here :)